libs/blocks/nav-marquee/nav-marquee.css

```diff
@@ -0,0 +1,278 @@
+.nav-marquee {
+  flex: 1 1 auto;
+}
+
+.nav-marquee.light {
+  color: var(--text-color);
+  background-color: #FFF;
+}
+
+.nav-marquee.dark {
+ background-color: #000;
+}
+
+.nav-marquee .content-table {
+  border: 1px solid #E8E8E8;
+  background-color: #FAFAFA;
+}
+
+.content-table .offer-container {
+  flex: 1 1 auto; 
+  padding: var(--spacing-s);
+}
+
+.content-table .offer-title {
+  font-size: var(--type-heading-xs-size);
+  margin: 0;
+  font-weight: bold;
+}
+
+.content-table .offer-list {
+  display: flex;
+  flex-direction: column; 
+  gap: var(--spacing-s);
+  margin-top: var(--spacing-s);
+  margin-bottom: 0;
+  padding: 0;
+}
+
+.content-table .offer-item {
+  min-width: 200px;
+  display: flex;
+  position: relative; 
+  justify-content: space-between;
+  background-color: #FFF;
+  border: 1px solid #4B4B4B;
+  border-radius: 4px;
+  padding: var(--spacing-s);
+  cursor: pointer;
+  line-height: 1.5;
+}
+
+.content-table .offer-item:hover {
+  box-shadow: 2px 4px 5px #0000001A;
+  border: 1px solid #1473E6;
+}
+
+.content-table .offer-item:focus-within {
+  border: 1px solid #1473E6;
+  background-color: #F5F9FD;
+}
+
+.nav-marquee .marquee p {
+  margin: 0 0 var(--spacing-s);
+}
+
+.nav-marquee .marquee .foreground .icon-area img {
+  height: 20px;
+  float: left;
+  padding-right: 16px;
```

Comment:
This looks like a good place for a spacing token `--spacing-xs` and float feels kind of outdated. Consider using flex-box w/ gap